app/data_handler.py

```diff
@@ -78,21 +94,20 @@ def importHostsToDB(self, csvImport):
                 local_creds = False
 
             try:
-                host = app.models.Host(hostname=row[0], ipv4_addr=row[1],
+                # TODO could probably use self.addHostToDB
+                host = app.models.Host(hostname=row[0].strip(),
+                                       ipv4_addr=row[1],
                                        type=row[2].capitalize(),
                                        ios_type=ios_type,
                                        local_creds=local_creds)
                 app.db.session.add(host)
                 app.db.session.flush()
+                app.db.session.commit()
```

Comment:
as a consideration, we should probably limit committing to the database for each host, and should just batch them all up and do it once.

Reply:
Good suggestion.  Added to this PR.